robot_skills/src/robot_skills/mockbot.py

```diff
@@ -339,14 +351,10 @@ def __init__(self, *args, **kwargs):
         # Body parts
         self.add_body_part('base', Base(self.robot_name, self.tf_listener))
         self.add_body_part('torso', Torso(self.robot_name, self.tf_listener, self.get_joint_states))
-        self.add_arm_part(
-            'leftArm',
-            Arm(self.robot_name, self.tf_listener, self.get_joint_states, "left")
-        )
-        self.add_arm_part(
-            'rightArm',
-            Arm(self.robot_name, self.tf_listener, self.get_joint_states, "right")
-        )
+
+        self.add_arm_part('leftArm', Arm(self.robot_name, self.tf_listener, self.get_joint_states, "left"))
+        self.add_arm_part('rightArm', Arm(self.robot_name, self.tf_listener, self.get_joint_states, "right"))
```

Comment:
you can just use robot_name instead of self.robot_name right?

Reply:
we could, but it doesnt make much of a difference